docs/UserGuide.md

```diff
@@ -362,7 +359,6 @@ Format: `add -l n/LESSON_NAME [c/LESSON_CAPACITY] [p/LESSON_PRICE] [t/LESSON_TIM
 
 * The lesson's capacity, price and timing are optional details for you to include.
 * The lesson name should only contain alphanumeric characters and spaces.
-* Duplicate lessons cannot be added. A lesson counts as a duplicate if it has the same lesson name as an existing lesson. Letter case is ignored when determining if two lesson names are the same.
```

Comment:
Sorry if this is a blur qn but why did u remove this haha

Reply:
Oh i removed it as this is already mentioned in lines 369 and 370